Library/Homebrew/cmd/list.rb

```diff
@@ -76,7 +76,8 @@ def list
     lib/ruby/gems/[12].*
     lib/ruby/site_ruby/[12].*
     lib/ruby/vendor_ruby/[12].*
-    manpages/brew.1
+    manpages/man1/brew.1
+    manpages/man1/brew-cask.1
```

Comment:
@joshka Ping on this, let's leave it as-is for now, please and save moving these around to another PR.